lighthouse-core/config/default-config.js

```diff
@@ -521,7 +527,11 @@ const defaultConfig = {
         {id: 'document-title', weight: 3, group: 'a11y-names-labels'},
         {id: 'duplicate-id-active', weight: 3, group: 'a11y-navigation'},
         {id: 'duplicate-id-aria', weight: 10, group: 'a11y-aria'},
+        /** TODO: add weight for v8 */
+        {id: 'empty-table-header', weight: 0, group: 'a11y-tables-lists'},
```

Comment:
```suggestion
        {id: 'empty-table-header', weight: 2, group: 'a11y-tables-lists'},
```
to match `heading-order`? but it's a "SHOULD" and has impact "Minor", so I guess you could even argue a weight of 1...